quinn-proto/src/shared.rs

```diff
@@ -1,9 +1,7 @@
 use std::{fmt, net::SocketAddr, time::Instant};
 
-use bytes::{Buf, BufMut, BytesMut};
-use rand::Rng;
-
 use crate::{coding::BufExt, packet::PartialDecode, ResetToken, MAX_CID_SIZE};
+use bytes::{Buf, BufMut, BytesMut};
```

Comment:
`use` statements referring to external items should be in a separate block, preceding those referring to internal ones.